schema/properties/bezier-keyframe.json

```diff
@@ -11,7 +11,10 @@
                 "s": {
                     "title": "Value",
                     "description": "Value at this keyframe.",
-                    "$ref": "#/$defs/values/bezier"
+                    "type": "array",
```

Comment:
Ye I think I forgot to add the array, we should specify the array needs to contain exactly one item

Reply:
Good idea, done.